python/tvm/script/builder/tir/prim_func_frame.py

```diff
@@ -40,3 +41,73 @@ def arg(name, obj) -> Union[Var, Buffer]:
 
 
 setattr(prim_func, "dispatch_token", "tir")
+
+
+def func_attr(attrs) -> DictAttrs:
+    return _ffi_api.FuncAttrs(
+        make_node("DictAttrs", **attrs)
+    )  # pylint: disable=no-member # type: ignore
```

Comment:
see my comment here: https://github.com/junrushao1994/tvm/pull/42/files#r895044696.

after this change, we only need to do:

```suggestion
def func_attr(attrs: Dict[str, Any]) -> None:
    return _ffi_api.FuncAttrs(attrs)  # pylint: disable=no-member # type: ignore
```